tensorflow/compiler/xla/service/gpu/gpu_fusible.cc

```diff
@@ -150,8 +150,7 @@ bool IsLoopFusible(const HloInstruction& instr) {
          instr.opcode() == HloOpcode::kConcatenate ||
          instr.opcode() == HloOpcode::kDynamicSlice ||
          instr.opcode() == HloOpcode::kDynamicUpdateSlice ||
-         (instr.opcode() == HloOpcode::kFusion &&
-          instr.fusion_kind() == HloInstruction::FusionKind::kLoop) ||
+         instr.opcode() == HloOpcode::kFusion ||
```

Comment:
Existing input fusions are not loop-fusible, hence we should return false as we did before. Please update IsInputFusible as suggested in my other comment instead.

Reply:
What about output and custom fusion nodes? Should they also be disallowed from loop fusion?

Reply:
Yes, that's a good point. We want to check for FusionKind::kLoop here.